tests/test_ote_training.py

```diff
@@ -383,8 +381,20 @@ class TestOTETraining:
     PERFORMANCE_RESULTS = None # it is required for e2e system
 
     DEFAULT_NUM_ITERS = 1
+    DEFAULT_BATCH_SIZE = 2
+    SHORT_TEST_PARAMETERS_NAMES_FOR_GENERATING_ID = OrderedDict([
+            ('model_name', 'model'),
+            ('dataset_name', 'dataset'),
+            ('num_training_iters', 'num_iters'),
+            ('batch_size', 'batch'),
+            ('usecase', 'usecase'),
+    ])
+    TEST_PARAMETERS_DEFINING_IMPL_BEHAVIOR = ('model_name', 'dataset_name', 'num_training_iters')
```

Comment:
Please add a comment explaining the semantics of this tuple.

Reply:
Ok

Reply:
Done